workflows/scrnaseq.nf

```diff
@@ -37,9 +37,14 @@ workflow SCRNASEQ {
         error "Only cellranger supports `protocol = 'auto'`. Please specify the protocol manually!"
     }
 
-    params.fasta = getGenomeAttribute('fasta')
-    params.gtf = getGenomeAttribute('gtf')
-    params.star_index = getGenomeAttribute('star')
+    // collect paths from genome attributes file (e.g. iGenomes.config; optional)
+    params.fasta            = getGenomeAttribute('fasta')
+    params.gtf              = getGenomeAttribute('gtf')
+    params.star_index       = getGenomeAttribute('star')
+    params.salmon_index     = getGenomeAttribute('simpleaf')
+    params.txp2gene         = getGenomeAttribute('simpleaf_tx2pgene')
+    params.cellranger_index = getGenomeAttribute('cellranger')
+    params.cellranger_index = getGenomeAttribute('cellrangerarc')
```

Comment:
Wouldn't this always override with the `cellrangerarc` attribute?

Reply:
Well spotted. How about we make this overriding conditional?

```nextflow
    // Make cellranger-arc index overwriting conditional
    if (params.aligner == "cellrangerarc") {
        params.cellranger_index = getGenomeAttribute('cellrangerarc')
    }
```

Reply:
Or how about in this way?
```
// Make cellranger or cellranger-arc index overwriting conditional
if (params.aligner in ["cellranger", "cellrangermulti"]){
        params.cellranger_index = getGenomeAttribute('cellranger')
    } else if (params.aligner == "cellrangerarc"){
        params.cellranger_index = getGenomeAttribute('cellrangerarc')
    }
    ch_cellranger_index = params.cellranger_index ? file(params.cellranger_index) : []
```

Reply:
agreed, this looks cleaner. I have added a new commit.